permissions/plugin-abap-ci.yml

```
@@ -0,0 +1,7 @@
+---
+name: "abap-ci"
+github: "jenkinsci/abap-ci-plugin"
+paths:
+- "org/jenkins-ci/plugins/abap-ci"
```

```suggestion
- "io/jenkins/plugins/abap-ci"
```

See https://github.com/jenkinsci/abap-ci-plugin/blob/master/pom.xml#L10